modules/actions/circularize.js

```diff
@@ -128,12 +128,10 @@ export function actionCircularize(wayId, projection, maxAngle) {
                 ]);
 
                 // choose a nearnode to use as the original
-                var min = Infinity;
                 for (var nodeId in nearNodes) {
                     var nearAngle = nearNodes[nodeId];
                     var dist = Math.abs(nearAngle - angle);
-                    if (dist < min) {
-                        dist = min;
```

Comment:
haha this should probably be `min = dist;`.  I always wondered why this code didn't work.

Reply:
Ah :) Will alter

Reply:
Surely the assignment isn't necessary atol, given that `min` is never used?

Reply:
`min` is used each time through the loop.  This loop is trying to find the nearest existing node for a newly added node to appear next to.  This was intended to be a cool effect that occurs [during the few milliseconds of transition when a feature is circularized](https://github.com/openstreetmap/iD/issues/3659#issuecomment-269004422).  Because I had the assignment backwards, instead all the newnodes pop out from the "first" node of the circle and then just work their way around the circle to where they should be.  I'm probably the only person who ever noticed this.

Reply:
Oh sorry, I thought the `min` was declared inside the loop :)
Yeah, that makes sense